sdk/rust/oak/src/lib.rs

```diff
@@ -462,3 +461,69 @@ pub fn run_event_loop<T: crate::io::Decodable, N: Node<T>>(mut node: N, in_handl
         }
     }
 }
+
+/// Register a new node entrypoint.
+///
+/// This registers the entrypoint name and the expression used to construct the
+/// node instance.
```

Comment:
Worth mentioning that what trait the node instance needs to implement?

Reply:
Good idea, added link to `Node` trait.